compiler/src/dotty/tools/dotc/transform/PCPCheckAndHeal.scala

```diff
@@ -119,8 +119,7 @@ class PCPCheckAndHeal(@constructorOnly ictx: Context) extends TreeMapWithStages(
           // Replace it with a properly encoded type splice. This is the normal for expected for type splices.
           tp.prefix.select(tpnme.splice)
         case tp: NamedType =>
-          if (tp.prefix.isInstanceOf[TermRef] && tp.prefix.isStable) tp
-          else checkSymLevel(tp.symbol, tp, pos) match {
+          checkSymLevel(tp.symbol, tp, pos) match {
```

Comment:
What if we just change change test to `tp.symbol.isStatic`?

Reply:
The checkSymLevel already check that